plugins/search/indexer/indexer-java/src/cc/search/indexer/app/Indexer.java

```diff
@@ -82,6 +83,8 @@ public class Indexer implements AutoCloseable, IndexerService.Iface {
   private Indexer(Options options_) throws IOException {
     _options = options_;
 
+    FileLoggerInitializer.addFileOutput(_options, _log, "indexPlugin");
```

Comment:
In the current version the last parameter is "indexer".
This means that the logs for it will go to a "<ProjectName>_indexer.log" file in the given directory.
Should this change?

Reply:
I took another look at it, and I think it's okay to leave it like this.